.github/CODEOWNERS

```diff
@@ -52,6 +52,109 @@ src/applications/find-forms @department-of-veterans-affairs/vsa-public-websites-
+src/site/stages/build/process-cms-exports/schemas/output/taxonomy_term-products.js
+src/site/stages/build/process-cms-exports/schemas/output/taxonomy_term-topics.js
+src/site/stages/build/process-cms-exports/transformers/block_content-promo.js
+src/site/stages/build/process-cms-exports/transformers/node-basic_landing_page.js
+src/site/stages/build/process-cms-exports/transformers/node-checklist.js
+src/site/stages/build/process-cms-exports/transformers/node-faq_multiple_q_a.js
+src/site/stages/build/process-cms-exports/transformers/node-full_width_banner_alert.js
+src/site/stages/build/process-cms-exports/transformers/node-landing_page.js
+src/site/stages/build/process-cms-exports/transformers/node-media_list_images.js
+src/site/stages/build/process-cms-exports/transformers/node-media_list_videos.js
+src/site/stages/build/process-cms-exports/transformers/node-q_a.js
+src/site/stages/build/process-cms-exports/transformers/node-step_by_step.js
+src/site/stages/build/process-cms-exports/transformers/node-support_resources_detail_page.js
+src/site/stages/build/process-cms-exports/transformers/node-support_service.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-alert_single.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-audience_topics.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-button.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-contact_information.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-email_contact.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-link_teaser.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-list_of_link_teasers.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-lists_of_links.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-process.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-q_a.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-q_a_group.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-react_widget.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-rich_text_char_limit_1000.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-step.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-step_by_step.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-table.js
+src/site/stages/build/process-cms-exports/transformers/paragraph-wysiwyg.js
+src/site/stages/build/process-cms-exports/transformers/taxonomy_term-administration.js
+src/site/stages/build/process-cms-exports/transformers/taxonomy_term-audience_beneficiaries.js
+src/site/stages/build/process-cms-exports/transformers/taxonomy_term-lc_categories.js
+src/site/stages/build/process-cms-exports/transformers/taxonomy_term-products.js
+src/site/stages/build/process-cms-exports/transformers/taxonomy_term-topics.js
```

Comment:
Can we break up the transformers into subfolders to make code ownership easier to manage?

Reply:
I like the idea of that, but at the same time our current team structure will likely change into the future so I wonder if it will make it for future teams to find a transformer if we do that. What do you think?

Reply:
I still think grouping the transformers in some way would make sense. We have a similar issue with the liquid templates.